openpgp/packet/opaque_test.go

```diff
@@ -51,7 +51,7 @@ func TestOpaqueParseReason(t *testing.T) {
 	const expectedBad = 3
 	// Test post-conditions, make sure we actually parsed packets as expected.
 	if badPackets != expectedBad {
-		t.Errorf("unexpected # unparseable packets: %d (want %d)", badPackets, expectedBad)
+		t.Errorf("unexpected # unparsable packets: %d (want %d)", badPackets, expectedBad)
```

Comment:
I think in CS contexts "unparseable" is more common than "unparsable", and [wikitionary](https://en.wiktionary.org/wiki/unparseable) lists it as an alternative spelling, so I would leave this one as-is :)
```suggestion
		t.Errorf("unexpected # unparseable packets: %d (want %d)", badPackets, expectedBad)
```

Reply:
Ok, changed.